src/pages/SegmentAssignments.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Heading } from '@chakra-ui/react';
+import { OCHBackend } from '../common/utils';
+
+import { SectionSelector, SegmentAssignmentTable } from '../components/SegmentAssignmentsTable';
+
+const sortSections = sectionsArray => {
+  const compare = (a, b) => {
+    if (a._id < b._id) return -1;
+    if (a._id > b._id) return 1;
+    return 0;
+  };
+  return sectionsArray.sort(compare);
+};
+
+const SegmentAssignments = () => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'isLoading' is assigned a value but never used.